tests/nnstreamer_datarepo/unittest_datareposrc.cc

```diff
@@ -326,7 +325,7 @@ TEST (datareposrc, readAudioRaw)
   GMainLoop *loop;
   GCallback handler = G_CALLBACK (new_data_cb);
   const gchar *str_pipeline
-      = "datareposrc location=audio1.raw json=audio1.json ! tensor_converter ! tensor_sink name=tensor_sink0";
+      = "datareposrc location=audio1.raw json=audio1.json ! tensor_converter ! tensor_sink name=tensor_sink0 sync=true";
```

Comment:
add both ```sync=false``` and ```sync=true``` case. Do not remove previous cases, but add new cases.

Reply:
Thank you. I will add new TC